go/page.templ

```diff
@@ -45,3 +69,24 @@ templ PageSection(p props.PageSection) {
 		</div>
 	</div>
 }
+
+// PageBreadcrumbs provides a popui component to show navigation breadcrumbs. List items can be either links or text
+templ PageBreadcrumbs() {
+	<ul class={ classes.PageBreadcrumbs }>
+		{ children... }
+	</ul>
+}
+
+// PageActions provides a wrapper for displaying items such as buttons at the right of a PageHeader component
+templ PageActions() {
+	<div class={ classes.PageHeaderActions }>
+		{ children... }
+	</div>
+}
```

Comment:
Are these still needed?

Reply:
I'd say we can keep it in case, for some reason, we want to use them outside a header?